src/cloud-element-templates/cmd/ElementTemplatesCommands.js

```diff
@@ -0,0 +1,66 @@
+import ChangeElementTemplateHandler from './ChangeElementTemplateHandler';
+import RemoveElementTemplateHandler from '../../element-templates/cmd/RemoveElementTemplateHandler';
+import MultiCommandHandler from '../../element-templates/cmd/MultiCommandHandler';
+
+export default class ElementTemplatesCommands {
```

Comment:
Please rename this to `ElementTemplateCommands`

Reply:
Done. ✅